.github/CHANGELOG.md

```diff
@@ -185,6 +185,16 @@
 
 <h3>Improvements</h3>
 
+* The internal variables ``All`` and ``Any`` to mark a gate to act on all or any 
+  wires were refactored to ``AllWires`` and ``AnyWires`` and their class to
+  ``ActOn``. 
+  [(#614)](https://github.com/XanaduAI/pennylane/pull/614)
+
+  For example, ``AllWires`` can now be 
+  imported via
+  
+  >>> qml.operations.ActOn.AllWires
```

Comment:
This is a very minor nitpick, but should this go in breaking changes rather than improvements?

Reply:
I thought the breaking changes only concern the user interface? If not, then you are right. Will move.